packages/ragbits-document-search/examples/documents_chat.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
```

Comment:
You can use `uv` script syntax here to indicate what dependencies this script has (take a look on other examples, or files in `scripts/`)

Reply:
https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies